src/test/java/com/iabtcf/encoder/field/FieldEncoderMapTest.java

```diff
@@ -0,0 +1,57 @@
+        Assert.assertEquals(FIELD_ENCODER_MAP.cmpId, getValueOfFieldMap(Fields.cmpId));
+        Assert.assertEquals(FIELD_ENCODER_MAP.cmpVersion, getValueOfFieldMap(Fields.cmpVersion));
+        Assert.assertEquals(FIELD_ENCODER_MAP.consentScreen, getValueOfFieldMap(Fields.consentScreen));
+        Assert.assertEquals(FIELD_ENCODER_MAP.consentLanguage, getValueOfFieldMap(Fields.consentLanguage));
+        Assert.assertEquals(FIELD_ENCODER_MAP.vendorListVersion, getValueOfFieldMap(Fields.vendorListVersion));
+        Assert.assertEquals(FIELD_ENCODER_MAP.policyVersion, getValueOfFieldMap(Fields.policyVersion));
+        Assert.assertEquals(FIELD_ENCODER_MAP.isServiceSpecific, getValueOfFieldMap(Fields.isServiceSpecific));
+        Assert.assertEquals(FIELD_ENCODER_MAP.useNonStandardStacks, getValueOfFieldMap(Fields.useNonStandardStacks));
+        Assert.assertEquals(FIELD_ENCODER_MAP.specialFeatureOptIns, getValueOfFieldMap(Fields.specialFeatureOptIns));
+        Assert.assertEquals(FIELD_ENCODER_MAP.purposeConsents, getValueOfFieldMap(Fields.purposeConsents));
+        Assert.assertEquals(FIELD_ENCODER_MAP.purposeLegitimateInterest,
+                getValueOfFieldMap(Fields.purposeLegitimateInterest));
+        Assert.assertEquals(FIELD_ENCODER_MAP.purposeOneTreatment, getValueOfFieldMap(Fields.purposeOneTreatment));
+        Assert.assertEquals(FIELD_ENCODER_MAP.publisherCountryCode, getValueOfFieldMap(Fields.publisherCountryCode));
+        Assert.assertEquals(FIELD_ENCODER_MAP.vendorConsents, getValueOfFieldMap(Fields.vendorConsents));
+        Assert.assertEquals(FIELD_ENCODER_MAP.vendorLegitimateInterest,
+                getValueOfFieldMap(Fields.vendorLegitimateInterest));
+        Assert.assertEquals(FIELD_ENCODER_MAP.publisherRestrictions, getValueOfFieldMap(Fields.publisherRestrictions));
+        Assert.assertEquals(FIELD_ENCODER_MAP.segmentType, getValueOfFieldMap(Fields.segmentType));
+        Assert.assertEquals(FIELD_ENCODER_MAP.vendorsDisclosed, getValueOfFieldMap(Fields.vendorsDisclosed));
+        Assert.assertEquals(FIELD_ENCODER_MAP.vendorsAllowed, getValueOfFieldMap(Fields.vendorsAllowed));
+        Assert.assertEquals(FIELD_ENCODER_MAP.publisherConsents, getValueOfFieldMap(Fields.publisherConsents));
+        Assert.assertEquals(FIELD_ENCODER_MAP.publisherLegitimateInterest,
+                getValueOfFieldMap(Fields.publisherLegitimateInterest));
+        Assert.assertEquals(FIELD_ENCODER_MAP.numCustomPurposes, getValueOfFieldMap(Fields.numCustomPurposes));
+        Assert.assertEquals(FIELD_ENCODER_MAP.publisherCustomConsents,
+                getValueOfFieldMap(Fields.publisherCustomConsents));
+        Assert.assertEquals(FIELD_ENCODER_MAP.publisherCustomLegitimateInterest,
+                getValueOfFieldMap(Fields.publisherCustomLegitimateInterest));
+    }
+
+    @Test
+    public void whenMapKeyIsInvalidThenTheValueShouldIsNull() {
+        Assert.assertNull(getValueOfFieldMap(INVALID_KEY));
```

Comment:
Is this not the default language feature?  If it is then I don't think we need to test it...

Reply:
It's true, I will remove.